### PR TITLE
creating the buyer html and css pages

### LINKED ISSUE #1: Buyer General HTML + CSS Structure
write basic HTML elements for the buyer page including:
1- div for products
2- navbar
3- buttons to move to other pages
write CSS to style the page generally to look good to use:
- center the products in div
- style navbar
- style buttons
- use flexbox

### PR DESCRIPTION
**Pull request by AlaaShurrab**
creating the buyer html and css pages
adding basic style

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>code SOZAbit</title>
+    <link rel="stylesheet" href="styles/buyer.css">
+</head>
+<body>
+    <div class="navBar">
+        <div class="logo">
+            <!-- add logo here -->
+            <img src="#" alt="LOGO" class="logo">
+        </div>
```
Comment (zeinjendeya) [inline index.html]: use nav instead of div

File: index.html [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+        <div class="test"></div>
+        <div class="test"></div>
+        <div class="test"></div>
+        <div class="test"></div>
+        <div class="test"></div>
+
+    </div>
+    <div class="prototype">
+        <!-- seller here -->
+        <a href="#"></a>
```
Comment (zeinjendeya) [inline index.html]: add index.html to the link

File: styles/buyer.css [hunk 3]
```diff
@@ -0,0 +1,86 @@
+*{
+    padding: 0%;
+    margin: 0%;
+}
+ html{
+     font-size: 16px;
+ }
+
+
+.navBar{
```
Comment (zeinjendeya) [inline styles/buyer.css]: change rulset to nav after changing the div element in HTML file

**Review by Sondos-Alalami — APPROVED** [review]

**Review by zeinjendeya — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (AlaaShurrab) [reply index.html]: done

File: index.html (hunk 2 above)
Comment (AlaaShurrab) [reply index.html]: done

**Review by Osama-AlAttar — APPROVED** [review]

File: styles/buyer.css (hunk 3 above)
Comment (AlaaShurrab) [reply styles/buyer.css]: DONE

**Review by zeinjendeya — APPROVED** [review]

**Pull request merged**